Cargo.toml

```diff
@@ -17,8 +17,8 @@ azure-devops = { project = "jonhoo/jonhoo", pipeline = "tsunami", build = "25" }
 maintenance = { status = "passively-maintained" }
 
 [features]
-default = ["aws", "azure", "baremetal"]
-aws = ["rusoto_core", "rusoto_ec2", "futures-util", "tempfile", "ubuntu-ami"]
+default = ["aws", "azure", "baremetal", "openssh"]
```

Comment:
Worth sticking a blurb about this in the crate documentation?

In particular, you could give an example or two of when it's useful :)

Reply:
Also `Tsunami::connect_all` maybe deserves a note

Reply:
> Maybe connect or initialize and then you can have a private connect_ssh (what you call do_connect_ssh now)?

I've renamed `connect_ssh` to `connect`.

> Worth sticking a blurb about this in the crate documentation?
> 
> In particular, you could give an example or two of when it's useful :)

Could you say something more about this? I've already added some document about these cargo features.

> Also `Tsunami::connect_all` maybe deserves a note

Good point. I believe the only reason to call `connect_all` without `openssh` is to get to `Machine`s.